demos/wizard.php

```diff
@@ -36,18 +32,19 @@ public function dbConnectFromWizard()
     }
 }
 
-$app = new AppWizard(); // App without authentication to be able to freely import data
+$app = new AppWizard(['title' => 'Agile Toolkit - Wizard setup']); // App without authentication to be able to freely import data
 $app->initLayout('Centered');
 
 /** @var Wizard $wizard */
-$wizard=$app->add('Wizard');
+$wizard = $app->add('Wizard');
```

Comment:
In the other files, you used ::addTo(). Would be great to have it here too to have demo code consistent